src/com/cft/contactmerge/contact/Name.java

```diff
@@ -87,6 +78,6 @@ public AnswerType isMatch(Name otherProperty) {
     }
 
     public String toString() {
-        return getFirstName() + ", " + getMiddleName() + ", " + getPrefix() + ", " + getLastName() + ", " + getSuffix();
+        return getFirstName() + ", " + getMiddleName() + ", " + getLastName() + ", " + getPrefix() + ", " + getSuffix();
```

Comment:
Sorry... Missed seeing this in the original review. Many/most contacts in ITK  won't have a middle name, prefix, or suffix. I imagine this will be common with other customers as well. If those values are empty we don't want to show the extra commas. I don't even think we have to show those values though because toString is only suppose to show a reasonable summary of the object's value. So... "FirstName LastName" is fine. I still prefer "LastName, FirstName" but just my preference.

Reply:
nice catch!